pkg/riff/commands/riff_test.go

```diff
@@ -33,3 +37,43 @@ func TestRiffCommand(t *testing.T) {
 
 	table.Run(t, commands.NewRiffCommand)
 }
+
+func TestRiffSubCommands(t *testing.T) {
```

Comment:
This test is fragile and not really necessary. We know which commands are registered because we have CI generate markdown docs from each command. These docs are committed to the repository and any drift between the generated and committed content will fail CI.

https://github.com/projectriff/cli/blob/e5a90713536175ecedaece277e0a98df49ca636e/Makefile#L44-L46

https://github.com/projectriff/cli/blob/e5a90713536175ecedaece277e0a98df49ca636e/azure-pipelines.yml#L21